apis/handlers/deleteprog.go

```diff
@@ -0,0 +1,68 @@
+// Copyright Contributors to the L3AF Project.
+// SPDX-License-Identifier: Apache-2.0
+
+package handlers
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+
+	"io/ioutil"
+	"net/http"
+
+	"github.com/rs/zerolog/log"
+
+	"github.com/l3af-project/l3afd/kf"
+	"github.com/l3af-project/l3afd/models"
+)
+
+// DeleteEbpfProgram   remove Ebpf programs on node
+// @Summary removing Ebpf Programs on node
+// @Description remove Ebpf Programs on node
+// @Accept  json
+// @Produce  json
+// @Param cfgs body []models.DeleteApiValues true "BPF programs"
+// @Success 200
+// @Router /l3af/configs/v1/delete [post]
+func DeleteEbpfPrograms(ctx context.Context, kfcfg *kf.NFConfigs) http.HandlerFunc {
+
+	return func(w http.ResponseWriter, r *http.Request) {
+		mesg := ""
+		statusCode := http.StatusOK
+
+		w.Header().Add("Content-Type", "application/json")
+
+		defer func(mesg *string, statusCode *int) {
+			w.WriteHeader(*statusCode)
+			_, err := w.Write([]byte(*mesg))
+			if err != nil {
+				log.Warn().Msgf("Failed to write response bytes: %v", err)
+			}
+		}(&mesg, &statusCode)
+
+		bodyBuffer, err := ioutil.ReadAll(r.Body)
+		if err != nil {
+			mesg = fmt.Sprintf("failed to read request body: %v", err)
+			log.Error().Msg(mesg)
+			statusCode = http.StatusInternalServerError
+			return
+		}
+
+		var t []models.DeleteApiValues
+		if err := json.Unmarshal(bodyBuffer, &t); err != nil {
+			mesg = fmt.Sprintf("failed to unmarshal payload: %v", err)
+			log.Error().Msg(mesg)
+			statusCode = http.StatusInternalServerError
+			return
+		}
+
+		if err := kfcfg.DeleteeBPFPrograms(t); err != nil {
+			mesg = fmt.Sprintf("failed to DeleteeBPFPrograms : %v", err)
```

Comment:
```suggestion
		if err := kfcfg.DeleteEbpfPrograms(t); err != nil {
			mesg = fmt.Sprintf("failed to DeleteEbpfPrograms : %v", err)
```
for consistency with line 28

Reply:
done